vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps

```diff
@@ -0,0 +1,11 @@
+Composite: When I scroll to the end of the page
+When I scroll context to BOTTOM edge
+
+Composite: When I scroll to the start of the page
+When I scroll context to TOP edge
```

Comment:
risky change, there could be cases, when some context is set and user wants to scroll the page, but not context